Core/Core/Credentials/AccountManager.cs

```diff
@@ -266,7 +266,26 @@ public static void UpgradeAccount(string id)
   /// <returns></returns>
```

Comment:
Maybe we should update this summary to reflect these changes. Since we no longer "Get all accounts for a given server."

Reply:
agree!